RaspberryPi_README.md

```diff
@@ -0,0 +1,84 @@
+
+## Starting the Vuforia Spatial Edge Server on Raspberry Pi
+
+This tutorial explains the steps to start the Vuforia Spatial Edge Server on a Raspberry Pi. This toolbox has been tested for Ubuntu 20.04, Ubuntu 18.04 and the Raspian operating system on a Raspberry Pi 4 model B. To learn more about this toolbox refer to the [Vuforia Spatial Toolbox](https://spatialtoolbox.vuforia.com/) website.
+
+
+### Ubuntu:
+
+You can install Ubuntu on the Raspberry Pi using a MicroSD Card.
+First you need to download the OS image that will be written to the SD Card. 
+
+Different versions of Ubuntu 20.04 LTS and 18.04 can be found [here](https://ubuntu.com/download/raspberry-pi).
+
+In order to write the image of Ubuntu, you need to write the OS image onto the MicroSD Card. We used the Win32 Disk Imager softwre for windows. 
+You can find instructions for writing the OS image into the SD Card for each of the operating sytems here: 
+[Ubuntu](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-ubuntu#1-overview), 
+[MacOS](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-macos#1-overview), 
+[Windows](https://ubuntu.com/tutorials/create-an-ubuntu-image-for-a-raspberry-pi-on-windows#1-overview).
+
+After writing the image to the SD Card, insert the Card into your Raspberry Pi and plug in the power adapter. If you used the link above, the image is the Ubuntu Server and you need to install a desktop for it. 
+After downloading the specific version of Ubuntu for your raspberry Pi, the webiste will direct you to the instructions on installing the server and the desktop. Although the recommended desktops are xubuntu, lubuntu and kubuntu, we installed the main Ubuntu gnome desktop since our system has 4GB of RAM using the following command: 
+
+`$ sudo apt-get install ubuntu-desktop `
+
+
+
+### Raspian:
+
+You can install the Raspberry Pi Imager to write the OS image to your SD Card. The imager already has the the different Raspian images, so there is no need to download it. You can open the software and write the image. The imager can be downloaded from [here](https://www.raspberrypi.org/downloads/).
+ 
+Insert the SD Card into your Raspberry Pi and enter the Raspian enviornment. 
+
+
+#### The Following Steps are the same for Ubuntu and Raspian:
+
+When the OS is installed, make sure you install git: 
+
+`$ sudo apt install git`
+
+You can install Node.js v14.x by running the following commands:
+
+`$ curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash - `
```

Comment:
This should be formatted as the following (dropping the '\' )
```
\```
$ curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash - 
$ sudo apt-get install -y nodejs 
\```
```
Right now all the code blocks are on one line

Reply:
Oh is that for all the code snippets?

Reply:
Yep, the multiline code block with triple backticks is the correct thing to use in those places